dataprofiler/tests/profilers/utils.py

```diff
@@ -201,7 +202,7 @@ def assert_profiles_equal(actual, expected):
         elif isinstance(actual_value, dict):
             assert_profiles_equal(actual_value, expected_value)
         elif isinstance(actual_value, numbers.Number):
-            np.testing.assert_equal(actual_value, expected_value)
+            np.testing.assert_equal(actual_value, expected_value, f"{key}")
```

Comment:
clarity on failure, how i found the `num_negative issue`.

Reply:
nice